Utilities/Scripts/SlicerWizard/__version__.py

```diff
@@ -1,8 +1,8 @@
 __version_info__ = (
   5,
+  1,
   0,
-  0,
```

Comment:
It would be nice if the Slicer version would not need to be specified here. Is this really needed? I see there is a __version__.py.in - could that be used instead? Or the version could be stored in a simple CMake file (similarly to vtkVersion.cmake) that the Python script could parse?